src/cplus_plugin/gui/constant_raster_manager_dialog.py

```diff
@@ -0,0 +1,1135 @@
+            self.lbl_description.setText(
+                "<b>Description:</b> Constant rasters for NCS pathways"
+            )
+        elif label == "Activities":
+            self.sw_model_type_container.setCurrentIndex(1)
+            self._left_group.setTitle("Activities")
+            self.lbl_description.setText(
+                "<b>Description:</b> Constant rasters for activities"
+            )
+
+    def load_pathways(self):
+        """Load pathway metadata into the raster type dropdown and restore selection."""
+        # Get metadata for pathways
+        pathway_metadatas = self.constant_raster_registry.metadata_by_component_type(
+            ModelComponentType.NCS_PATHWAY
+        )
+        self.cbo_raster_type.clear()
+        for metadata in pathway_metadatas:
+            self.cbo_raster_type.addItem(metadata.display_name, metadata.id)
+
+        # Try to restore the previously selected pathway raster type
+        settings = QgsSettings()
+        settings.beginGroup("cplus/constant_rasters_dialog")
+        pathway_raster_type = settings.value("pathway_raster_type", None)
+        settings.endGroup()
+
+        # Select the saved raster type if available, otherwise first item
+        if pathway_raster_type:
+            for i in range(self.cbo_raster_type.count()):
+                if self.cbo_raster_type.itemData(i) == pathway_raster_type:
+                    self.cbo_raster_type.setCurrentIndex(i)
+                    break
+        elif self.cbo_raster_type.count() > 0:
+            self.cbo_raster_type.setCurrentIndex(0)
+
+        # Auto-select first checked pathway to load its values into widget
+        for row in range(self._pathways_model.rowCount()):
+            item = self._pathways_model.item(row)
+            if item and item.checkState() == QtCore.Qt.Checked:
+                index = self._pathways_model.indexFromItem(item)
+                self.lst_pathways.setCurrentIndex(index)
+                break
+
+    def on_pathway_type_selected(self, checked: bool):
+        """Slot raised when pathway radio button is selected."""
+        if not checked:
+            return
+
+        # Save previous state before switching
+        self._save_dialog_state()
+
+        self.update_component_type_labels("NCS Pathways")
+        self.load_pathways()
+
+    def load_activities(self):
+        """Load activity metadata into the raster type dropdown and restore selection."""
+        # Get metadata for activities
+        activity_metadatas = self.constant_raster_registry.metadata_by_component_type(
+            ModelComponentType.ACTIVITY
+        )
+        self.cbo_raster_type.clear()
+        for metadata in activity_metadatas:
+            self.cbo_raster_type.addItem(metadata.display_name, metadata.id)
+
+        # Try to restore the previously selected activity raster type
+        settings = QgsSettings()
+        settings.beginGroup("cplus/constant_rasters_dialog")
```

Comment:
@osundwajeff define these settings in conf.py. If GUI-related, consider using QgsGui utils.